curl_cffi/curl.py

```diff
@@ -422,10 +433,6 @@ def ws_send(self, payload: bytes, flags: CurlWsFlag = CurlWsFlag.BINARY) -> int:
         self._check_error(ret, "WS_SEND")
         return n_sent[0]
 
-    def ws_close(self) -> None:
```

Comment:
It wasn't used anywhere and didn't actually close the WebSocket. To properly close it, you have to send the close frame and close the curl handle with `curl_easy_cleanup()`.

Reply:
Yes, but they are different levels of APIs. The `Curl` class in on a lower level, while the `WebSocket` class is on a higher level. Plus, this function is `ws_close`, not `_ws_close`, although it's not used internally, it might be used externally by users.

Reply:
Okay, I added it back with a note on the limitations